src/main/java/com/greenfoxacademy/goddesstribesbackend/services/ResourceService.java

```diff
@@ -30,11 +28,23 @@ public Resource saveFoodAtStart(Townhall townhall) {
   }
 
   public Resource saveGoldAtStart(Townhall townhall) {
-    if (townhall != null){
+    if (townhall != null) {
```

Comment:
Please do add null checks only to controllers.

Reply:
Ok. I removed it.